internal/generate/crd/crd.go

```diff
@@ -371,6 +374,26 @@ func setCRDStorageVersion(crd *apiextv1beta1.CustomResourceDefinition) {
 	crd.Spec.Versions[0].Storage = true
 }
 
+func setSDKstamps(crd *apiextv1beta1.CustomResourceDefinition, operatorType string) {
```

Comment:
```suggestion
func (g Generator) setSDKstamps(crd *apiextv1beta1.CustomResourceDefinition) {
```
I think is better than pass the type.